config-models/stratum-1.x/README.md

```diff
@@ -0,0 +1,4 @@
+# Deprecated
+This directory contains the old structure of the Stratum configuration models.
```

Comment:
adding this file means you will have to update the version number in `Chart.yaml` for this directory